src/views/Home/components/Banners/FarmAuctionsBanner.tsx

```diff
@@ -9,6 +9,12 @@ const StyledSubheading = styled(Heading)`
   background-clip: text;
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
+  ${({ theme }) => theme.mediaQueries.xs} {
+    -webkit-text-stroke: 1px rgba(0, 0, 0, 0.5);
```

Comment:
Maybe text-shadow would be better? Also seems to have support for more browsers.

Reply:
If text-shadow is used, we need to use color instead of background therefore we will lose gradient. And the compliance code is a bit long that you can check here https://codepen.io/Dzorogh/pen/YPeMKd

Text-stroke is also supported by major browsers. https://caniuse.com/?search=text%20stroke